src/renderer/remote-loader.ts

```diff
@@ -136,23 +115,9 @@ export class RemoteLoader {
       const gist = await octo.gists.get({ gist_id: gistId });
       const values: Partial<EditorValues> = {};
 
-      // Add values for all default editors.
-      for (const editor of DEFAULT_EDITORS) {
-        values[editor] = this.getContentOrEmpty(gist, editor);
-      }
-
-      // Fetch any custom editor files that the user may have created in the gist.
-      const maybeCustomEditors = Object.keys(gist.data.files).filter(
-        (file) =>
-          !Object.values(DefaultEditorId).includes(file as DefaultEditorId),
-      );
-
-      // If it's a custom editor explicitly request permission before creation.
-      for (const mosaic of maybeCustomEditors) {
-        const verified = await this.verifyCreateCustomEditor(mosaic);
-        if (verified) {
-          values[mosaic] = this.getContentOrEmpty(gist, mosaic);
-          this.appState.customMosaics.push(mosaic as CustomEditorId);
+      for (const [id, data] of Object.entries(gist.data.files)) {
+        if (isKnownFile(id) || (await this.verifyCreateCustomEditor(id))) {
+          values[id] = data.content;
```

Comment:
Note: App.replaceFiddle(), which is called right after this via handleLoadingSuccess(), rebuilds the this.app.customMosaics array, so we no longer populate it here.

Reply:
The previous implementation iterates through all default editors, whereas this only iterates through gist files - I see above you say it gets rebuilt, so do you mean that the empty editors are appropriately populated there?

Reply:
Missing defaults will be hidden when the fiddle is loaded. If the user then goes and toggles on a default via the toolbar, it will be loaded with the electron-quick-start default via https://github.com/electron/fiddle/blob/d2b7b61afd81f3a1c6cf6d3260e5745be3e1e831/src/renderer/components/editor.tsx#L164

Reply:
ah, ok! cool :D